apps/extension/src/routes/popup/approval/tx-msg-handler.ts

```diff
@@ -10,7 +10,7 @@ export const isTxApprovalReq = (req: PopupRequest): req is TxApproval => {
 export const handleTxApproval: InternalMessageHandler<TxApproval> = (jsonReq, responder) => {
   useStore.setState(state => {
     state.txApproval.authorizeRequest = jsonReq.authorizeRequest;
-    state.txApproval.denomMetadataByAssetId = jsonReq.denomMetadataByAssetId;
+    state.txApproval.transactionViewFromPlan = jsonReq.transactionViewFromPlan;
```

Comment:
We're now translating the transaction plan into a transaction view inside the `authorize` impl [here](https://github.com/penumbra-zone/web/pull/378/files#diff-b2755ed7d656eafb2fa77b3ca380564a22ab97247124767e75f22fe233a191d7), so we have the `TransactionView` already once we're inside the popup. (Thus, we also no longer need `denomMetadataByAssetId` here, since it was previously being used to generate that `TransactionView` in the popup, but isn't any longer.)